doc/classes/CurveTexture.xml

```diff
@@ -14,10 +14,16 @@
 		<member name="curve" type="Curve" setter="set_curve" getter="get_curve">
 			The [code]curve[/code] rendered onto the texture.
 		</member>
+		<member name="mode" type="int" setter="set_texture_mode" getter="get_texture_mode" enum="CurveTexture.TextureMode" default="0">
```

Comment:
I'd name this `texture_mode` to match the enum. Also prevents risk of name conflicts with inherited properties.